PIL/WmfImagePlugin.py

```diff
@@ -153,7 +153,7 @@ def _load(self):
 
 
 def _save(im, fp, filename):
-    if _handler is None or not hasattr("_handler", "save"):
+    if _handler is None or not hasattr(_handler, "save"):
```

Comment:
OMG )

Reply:
You know something when you fix an "it could never work" bug that dates back to PIL.